md/dev/mediquette.txt

```diff
@@ -0,0 +1,74 @@
+
+Etiquette @medic
+
+Be kind to the plants and animals.
+
+Slack
+
+  1) Try to pay attention what channel you are writing in and keep noise
+     (tangential or unrelated discussion) to a minimum.  Use threads when
+     appropriate to keep tangential discussion contained and out of the main
+     channel.   Each channel should have a purpose with a description that
+     explains appropriate usage.
+
+  2) Never delete or edit someone else's content.  If you think something is
+     inappropriate then notify the person first and ask them politely to remove
+     it.
+
+  3) In general deleting something you already posted is poor etiquette.
+     Because it changes the context of the discussion.  If you have a problem
+     with something you wrote then just create a thread on it or just repost it
+     with your edits.
```

Comment:
It changes context which bothers me because then it potentially changes the meaning of someone's response... can potentially change the entire conversation, it just feels like dangerous territory, slippery slope that should be avoided or reserved for specific incidents.  I feel like editing and deleting posts needs some treatment here and shouldn't be left completely arbitrary.